src/Microsoft.AspNetCore.Routing/Template/TemplateBinder.cs

```diff
@@ -273,6 +277,26 @@ private string BindValues(UriBuildingContext context, RouteValueDictionary accep
             return context.ToString();
         }
 
+        private ParameterTransformer GetParameterTransformer(RoutePatternParameterPart parameterPart)
+        {
+            if (_parameterPolicyFactory == null)
+            {
+                return null;
+            }
+
+            for (int i = 0; i < parameterPart.ParameterPolicies.Count; i++)
+            {
+                // Use the first parameter transformer
+                var parameterPolicy = _parameterPolicyFactory.Create(parameterPart, parameterPart.ParameterPolicies[0]);
+                if (parameterPolicy is ParameterTransformer parameterTransformer)
+                {
+                    return parameterTransformer;
+                }
```

Comment:
Is there a problem with supporting multiple? Should we throw? What do you think is most reasonable?